src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,173 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+
+    private int currentAmount = 0;
+    private transient Node<T> first;
```

Comment:
what does "transient" do?

Reply:
if I got the point right, it means that this object contains not the certain object (static I'd say), but the link on the object, that (link) is changing during the program working

Reply:
it is used during serialization to ignore certain fields
https://www.geeksforgeeks.org/transient-keyword-java/

Reply:
thanks, now I know why it should be used :)
should I remove those for our project or let them stay?

Reply:
let's use only things we need